### ISSUE TITLE
Autoupload cannot be stopped

### ISSUE THREAD

**Issue by InfamousUser** [nextcloud/android#11671]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

1. Install Nextcloud app.
2. Connect to a server.
3. Configure a folder for autoupload.

### Expected behaviour

Files should start uploading, disabling the autoupload folder and/or cancelling the uploads (via the X in the upload menu) should cancel the pending uploads.

### Actual behaviour

X in the uploads menu does not clear the pending uploads, nor does disabling the autoupload folder. Force closing the app also doesn't cancel them (they are still there on app reopen). No way to stop the upoads anymore!

### Android version

13

### Device brand and model

Pixel 7 Pro

### Stock or custom OS?

Stock

### Nextcloud android app version

3.25.0 RC1

### Nextcloud server version

25.0.7

### Using a reverse proxy?

No

**Comment by tilllt** [comment]
To add to the misry of the Auto-upload implementation: if you had added an auto-upload folder with the wrong target and it added hundreds of files, you cannot clear the upload queue, rescan and upload to the right location. You can just watch and see how one file after another is uploaded to the wrong folder and not stop it.

**Comment by InfamousUser** [comment]
Yes, that is exactly how it happened! But I have to correct you - the autoupload has been broken for years. I've been trying to help get this fixed but nothing ever came of it.

**Comment by tautomer** [comment]
I think the same thing happens to manual uploads as well. The only thing to clear the queue is to clear the data of the app or uninstall/install the app.

**Comment by Croydon** [comment]
> I think the same thing happens to manual uploads as well. The only thing to clear the queue is to clear the data of the app or uninstall/install the app.

I can second that. I tried to manually upload ~60 pictures --- 4 weeks ago --- and I have still the upload queue stuck. I have long moved the pictures to my desktop PC and upload it there in Nextcloud. So the files do not even exist on my Android smartphone anymore.

I can't clear the entire queue and I can't delete individual queued items. I can only clear items that failed to upload. For some reason the app only tries to upload 2-5 files every few days so I am super stuck there.

I didn't dear to delete the app data yet, as I have other Nextcloud related apps and I'm not sure if they would be affected by that negatively.

**Comment by tautomer** [comment]
> I didn't dear to delete the app data yet, as I have other Nextcloud related apps and I'm not sure if they would be affected by that negatively.

I'm not entirely sure, since I don't use these apps personally. They will probably work fine after you reconfigure the nextcloud app settings as before.

**Comment by axodentally** [comment]
Problem still exists.

**Comment by cordvision** [comment]
Also have the exact same problem.. super annoying. I h ave 11031 files in the cue...

**Comment by divanke** [comment]
The problem still exists... Android 17, Nextcloud Version 34.1.0
